thefuck/shells/fish.py

```diff
@@ -127,3 +127,10 @@ def _put_to_history(self, command_script):
                     history.write(entry.encode('utf-8'))
                 else:
                     history.write(entry)
+
+    def can_edit(self):
+        return True
+
+    def edit_command(self, command):
+        """Return the shell editable command"""
+        return 'commandline -r "' + command + '"'
```

Comment:
we use string formatting rather than concatenation everywhere else

Reply:
What do you think about f-strings vs using `.format`?

Reply:
When were f-strings introduced?  Are we still supporting older versions of python3 or even python2 still?

Reply:
The CI system is still running on older python versions, so you can't use f-strings im afraid!

Reply:
https://travis-ci.org/github/nvbn/thefuck/builds/661796234?utm_source=github_status&utm_medium=notification

Reply:
Oops, that slipped through amidst the various experimentations I was doing. Sorry!